polyfills/CustomEvent/config.json

```diff
@@ -5,7 +5,7 @@
 	],
 	"browsers": {
 		"firefox": "6 - 10",
-		"ie": "9 - *",
+		"ie": "*",
```

Comment:
No, for convenience we define a baseline, so we ignore any range under the baseline.  Makes it easier to raise the baseline without having to adjust all the configs.